.nuspec/Xamarin.Forms.targets

```diff
@@ -54,6 +54,15 @@
 			Condition="'$(IsXBuild)' == 'true'" />
 	</Target>
 
+	<Target Name="_ValidateNETFrameworkVersion"
+			BeforeTargets="_CheckForInvalidConfigurationAndPlatform"
+			Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' AND '$(TargetFrameworkVersion.Substring(1))' &lt; '4.7' ">
```

Comment:
I think that it will be more clear to use `.TrimStart('vV')` instead `.Substring(1)`